src/V3Coverage.cpp

```diff
@@ -902,6 +905,22 @@ class CoverageVisitor final : public VNVisitor {
         lineTrack(nodep);
     }
 
+    // Currently unsupported for coverage expression
+    void visit(AstNodeFUniop* nodep) override { exprUnsupported(nodep); }
+    void visit(AstNodeFBiop* nodep) override { exprUnsupported(nodep); }
+    void visit(AstNodeFExpr* nodep) override { exprUnsupported(nodep); }
+    void visit(AstNodeFTaskRef* nodep) override { exprUnsupported(nodep); }
+
+    void exprUnsupported(AstNode* nodep) {
+        bool wasSeeking = m_seeking == SEEKING;
+        Objective oldSeeking = m_seeking;
+        if (wasSeeking) { abortExprCoverage(); }
+        m_seeking = ABORTED;
+        iterateChildren(nodep);
+        lineTrack(nodep);
+        if (!wasSeeking) { m_seeking = oldSeeking; }
+    }
```

Comment:
Please pass a string as to why to help future debug

```suggestion
    void exprUnsupported(AstNode* nodep, const string& why) {
        UINFO(9, "unsupported " << why << " " << nodep << endl);
        const bool wasSeeking = m_seeking == SEEKING;
        Objective oldSeeking = m_seeking;
        if (wasSeeking) abortExprCoverage();
        m_seeking = ABORTED;
        iterateChildren(nodep);
        lineTrack(nodep);
        if (!wasSeeking) m_seeking = oldSeeking;
    }
```